gluten-core/src/main/scala/org/apache/spark/sql/hive/HiveTableScanExecTransformer.scala

```diff
@@ -52,7 +52,7 @@ class HiveTableScanExecTransformer(
     relation: HiveTableRelation,
     partitionPruningPred: Seq[Expression])(session: SparkSession)
   extends HiveTableScanExec(requestedAttributes, relation, partitionPruningPred)(session)
-  with BasicScanExecTransformer {
+  with BaseScanTransformer {
```

Comment:
Better to still use `BasicScanExecTransformer` as this change is not necessary.

Reply:
OK. I will revert these changes first.